app/models/library_entry.rb

```diff
@@ -149,12 +149,9 @@ def activity
   end
 
   def kind
```

Comment:
what would happen if `kind` is `nil`? I don't think that is _actually_ possible but gonna throw it out there.